src/test/data/bitcoin-util-test.json

```diff
@@ -87,6 +87,14 @@
     "output_cmp": "txcreate2.json"
   },
   { "exec": "./bitcoin-tx",
+    "args": ["01000000000100000000000000000000000000"],
```

Comment:
Is this supposed to be parsed as a zero input, one output to `0x00` transaction?
```
01000000
00
01
  0000000000000000
  00
00000000
```

Wouldn't `010000000001000000000000` (which currently fails) be more suitable?
```
01000000
00
01
00
00
00000000
```

Following `01000000000000000000` behavior.
Unless there's another reason for using that one.